common/utils.py

```diff
@@ -0,0 +1,31 @@
+import string
+import random
+
+
+def generate_random_number(length):
+    rand_number = ''
+    for i in range(length):
+        rand_number += str(random.randint(0, 9))
+    return rand_number
+
+
+def create_word_from_number(number):
+    alphabet = string.ascii_lowercase
+    word = ''
+    while number:
+        letter_number = int(number[:2])
+        word += alphabet[letter_number % 26]
+        number = number[2:]
+    return word
+
+
+def generate_random_word(length):
+    alphabet = string.ascii_lowercase
+    return ''.join(random.choice(alphabet) for _ in range(length))
```

Comment:
Можно сократить код, так как переменная `alphabet` используется лишь один раз в данной функции.
```suggestion
    return ''.join(random.choice(string.ascii_lowercase) for _ in range(length))
```